plugins/modules/utm_proxy_frontend_info.py

```diff
@@ -93,7 +93,7 @@
       description: State of html rewrite.
       type: bool
     htmlrewrite_cookies:
-      description: Whether the html rewrite cookie will be set.
+      description: Whether the html rewrite cookie is set.
```

Comment:
```suggestion
      description: Whether the HTML rewrite cookie is set.
```
(Just make sure to not replace `.html` :) )

Reply:
there's a rule that picks words with space before them, we use it fro API, JSON, ...

BTW, I am thinking VM should also go in that bunch. Agree?

Reply:
Yep, VM should as well!